acme-cli/build.gradle

```diff
@@ -1,36 +1,30 @@
 plugins {
     id "groovy"
-    id "net.ltgt.apt-eclipse" version "0.21"
-    id "com.github.johnrengelman.shadow" version "5.0.0"
+    id "com.github.johnrengelman.shadow"
     id "application"
 }
 
-
-
-version "0.1"
-group "acme.cli"
-
-repositories {
-    mavenCentral()
-    maven { url "https://jcenter.bintray.com" }
-}
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
Is this necessary?
I thought including it by `id` was sufficient

Reply:
Yup it is 9d4158b